src/renderer/ui/components/generic/PlatformSpecific.js

```diff
@@ -1,14 +1,20 @@
 import { Component, PropTypes } from 'react';
+import os from 'os';
 
 export default class PlatformSpecific extends Component {
   static propTypes = {
     children: PropTypes.object,
     platform: PropTypes.string.isRequired,
+    version: PropTypes.string,
   };
 
   render() {
     if (process.platform === this.props.platform) {
-      return this.props.children;
+      if (!this.props.version) return this.props.children;
```

Comment:
@levrik I'm OK with overkill as long as it doesn't add too much file size to the overall package.  `semver` is like a few kb but it adds a lot more customizability.  I have an eventual plan to publish some of the internal React components of GPMDP as their own separate "Electron React Helpers" kind of package and using a standard like semver would be very good for if / when that happens